test-cases/longevity/longevity-large-partition-200k_pks-2days-i4i.yaml

```diff
@@ -29,6 +29,12 @@ stress_cmd: [
 "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=300 -clustering-row-count=200000 -clustering-row-size=uniform:10..10240 -partition-offset=1001 -concurrency=10 -connection-count=10 -consistency-level=quorum -rows-per-request=10 -timeout=90s",
              "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=300 -clustering-row-count=200000 -clustering-row-size=uniform:10..10240 -partition-offset=1301 -concurrency=10 -connection-count=10 -consistency-level=quorum -rows-per-request=10 -timeout=90s",
              "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=400 -clustering-row-count=200000 -clustering-row-size=uniform:10..10240 -partition-offset=1601 -concurrency=10 -connection-count=10 -consistency-level=quorum -rows-per-request=10 -timeout=90s",
+             # due to increasing of amount deleted rows/partitions (delete nemeses) we need to insert back the rows. Otherwise, the table may become empty after few deletions.
+             # Also, the first 250 partitions (as defined in partition_range_with_data_validation) won't be deleted. So it is not needed to be re-written
```

Comment:
I'm not sure about that.
Need to think of it.

On one hand, it's good that we have data that we re-write after it was deleted.
OTHO, it will be the entire data eventually.

Maybe, we want to re-write only one part and to add more partitions on beyond the original range.

Reply:
> I'm not sure about that. Need to think of it.
> 
> On one hand, it's good that we have data that we re-write after it was deleted. OTHO, it will be the entire data eventually.
> 
> Maybe, we want to re-write only one part and to add more partitions on beyond the original range.

There are load already defined in the test that add different types of partitions (not same as is inserted while prepare stage)

Reply:
test removed

Reply:
Changed it to re-write 2/3 of available partitions. 1/3 of partitions won't be re-written and will remain as tombstones after deletion